pandas/core/generic.py

```diff
@@ -1786,40 +1786,47 @@ def to_json(self, path_or_buf=None, orient=None, date_format=None,
                             index=index)
 
     def to_hdf(self, path_or_buf, key, **kwargs):
-        """Write the contained data to an HDF5 file using HDFStore.
+        """
+        Write the contained data to an HDF5 file using HDFStore.
+
+        Hierarchical Data Format (HDF) is self-describing, allowing an
+        application to interpret the structure and contents of a file with
+        no outside information. One HDF file can hold a mix of related objects
+        which can be accessed as a group or as individual objects.
+
+        In order to add another :class:`~pandas.DataFrame` or
+        :class:`~pandas.Series` to an existing HDF file please use append mode
+        and different a key.
 
         Parameters
         ----------
-        path_or_buf : the path (string) or HDFStore object
-        key : string
-            identifier for the group in the store
-        mode : optional, {'a', 'w', 'r+'}, default 'a'
-
-          ``'w'``
-              Write; a new file is created (an existing file with the same
-              name would be deleted).
-          ``'a'``
-              Append; an existing file is opened for reading and writing,
-              and if the file does not exist it is created.
-          ``'r+'``
-              It is similar to ``'a'``, but the file must already exist.
-        format : 'fixed(f)|table(t)', default is 'fixed'
-            fixed(f) : Fixed format
-                       Fast writing/reading. Not-appendable, nor searchable
-            table(t) : Table format
-                       Write as a PyTables Table structure which may perform
-                       worse but allow more flexible operations like searching
-                       / selecting subsets of the data
+        path_or_buf : str or pandas.HDFStore
+            File path or HDFStore object.
+        key : str
+            Identifier for the group in the store.
+        mode : {'a', 'w', 'r+'}, default is 'a'
+            Mode to open file:
+                - ``'w'``: write, a new file is created (an existing file with
+                    the same name would be deleted).
+                - ``'a'``: append, an existing file is opened for reading and
+                    writing, and if the file does not exist it is created.
+                - `'r+'`: similar to ``'a'``, but the file must already exist.
+        format : {'fixed', 'table'}, default is 'fixed'
+            Possible values:
+                - fixed: Fixed format. Fast writing/reading. Not-appendable,
```

Comment:
same here about indentation / blank line

Reply:
also, can you add single quotes around fixed (like `'fixed'`), to make it clear it is a string

(and same for table below)

Reply:
done